### PR TITLE
Docs: Add how-to for `PwCalculationTools.get_scf_accuracy`

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

File: docs/source/howto/calculations/pw.rst [hunk 1]
```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
...
+How to analyze the results
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a ``PwCalculation`` is completed, there are quite a few possible analyses to perform.
+
+How to check the SCF accuracy during the self-consistent cycle
+--------------------------------------------------------------
+
+During the self-consistent field cycle, the difference in energy of the newly computed charge density and the starting one is computed and stored.
+It can easily be retrieved through the :meth:`aiida_quantumespresso.tools.calculations.pw.PwCalculationTools.get_scf_accuracy` method.
```
Comment (mbercx) [inline docs/source/howto/calculations/pw.rst:325]: Can we somehow not show the entire path to the method?

Screenshot 2023-05-09 at 06 00 32

File: docs/source/howto/calculations/pw.rst [hunk 2]
```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
 
 .. |SSSP| replace:: Standard Solid-State Pseudopotentials (SSSP)
 .. _SSSP: https://www.materialscloud.org/discover/sssp/table/efficiency
+
+
+How to analyze the results
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a ``PwCalculation`` is completed, there are quite a few possible analyses to perform.
```
Comment (mbercx) [inline docs/source/howto/calculations/pw.rst:319]: This is not incorrect, but it is sort of funny that we say this and then only show one. 😉

**Review by mbercx — CHANGES_REQUESTED** [review]
Thanks @sphuber! Just left a few comments, but very clear in general.

File: docs/source/howto/calculations/pw.rst [hunk 3]
```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
...
+
+How to check the SCF accuracy during the self-consistent cycle
+--------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (sphuber) [reply docs/source/howto/calculations/pw.rst:322]: Ok, will change. If we merge this before the MyST rewrite, we shouldn't forget to merge this back in

File: docs/source/howto/calculations/pw.rst (hunk 1 above)
Comment (sphuber) [reply docs/source/howto/calculations/pw.rst:325]: yeah we can, just put `~` in front of it.

File: docs/source/howto/calculations/pw.rst (hunk 2 above)
Comment (sphuber) [reply docs/source/howto/calculations/pw.rst:319]: Sure, but I imagine we will be adding different analyses in the future. This is the first one to kick things off.

File: docs/source/howto/calculations/pw.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (mbercx) [reply docs/source/howto/calculations/pw.rst:322]: For sure, I'll convert and add it 👍

File: docs/source/howto/calculations/pw.rst [hunk 4]
```diff
@@ -311,3 +311,38 @@ To retrieve additional output files, specify the list of files in the ``ADDITION
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a ``PwCalculation`` is completed, there are quite a few possible analyses to perform.
+
+How to check the SCF accuracy during the self-consistent cycle
+..............................................................
+
+During the self-consistent field cycle, the difference in energy of the newly computed charge density and the starting one is computed and stored.
+It can easily be retrieved through the :meth:`~aiida_quantumespresso.tools.calculations.pw.PwCalculationTools.get_scf_accuracy` method.
+This method can be accessed directly through the node of a completed ``PwCalculation``:
```
Comment (mbercx) [inline docs/source/howto/calculations/pw.rst:326]: One final nitpick: I'd still mention the `tools` name space in this sentence, e.g.:

```suggestion
This method can be accessed directly through the ``tools`` of a completed ``PwCalculation`` node:
```

**Review by mbercx — DISMISSED** [review]
Just one final note, but I'll already approve. Thanks again!

**Pull request merged**